src/codegen/js/normalize.ts

```diff
@@ -0,0 +1,115 @@
+const SANITIZE_PREFIX = 'safe_'
+
+export function sanitizeKeyword(name: string) {
+  if (!reservedWord.includes(name)) return name
+
+  return SANITIZE_PREFIX + name
```

Comment:
Very minor but for consistency, it would be nice to use the same style as the rest of the codebase, i.e. 4 spaces and semicolons at the end of statements. (Maybe I should setup a formatter to automate this).

Reply:
I don't really care about the coding conventions, (apart from semicolons since automatic semicolon insertion in JS parsers is pretty terrible and has many pitfalls, and most JS/TS projects do use semicolons so standardJS despite its name is really not that common anymore). It's mostly a matter of what each developer is used to, however I think it's important to be able to adapt to the coding style of the project you're contributing to as it makes the PRs less noisy (you changed less lines than what is chown in the diff, but your formatter added noise).

So I setup a prettier config to make sure all files use the same conventions.